news-aggregation/persistence/NewsItemPersistence.ts

```diff
@@ -66,7 +96,8 @@ export class NewsItemPersistence {
 	//     associations.push(idAssociation);
 	// }
 
-	async getAllNewsById(allNews: NewsItem[]): Promise<object[]> {
+	// Question
+	public async getAllNewsById(allNews: NewsItem[]): Promise<object[]> {
```

Comment:
```suggestion
	public async getNewsByIds(Ids: string[]): Promise<NewsItem[]> {
```
Can pass the ids array instead.